decidim-core/app/controllers/decidim/searches_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Decidim
+  class SearchesController < Decidim::ApplicationController
+    include Rectify::ControllerHelpers
+    include FormFactory
+    include FilterResource
+    # include Orderable
+    include Paginable
+
+    helper Decidim::FiltersHelper
+    helper_method :term
+
+    def index
+      Search.call(term, current_organization, filters) do
+        on(:ok) do |results|
+          expose(results: results)
```

Comment:
What's this method? Is it https://github.com/andypike/rectify/blob/5d259cc3ac6e3c1fa10502790e8fd08b85c54aa7/lib/rectify/controller_helpers.rb#L18 ? Seems rather cryptic, as we're not using presenters here AFAIK, and it's only setting an instance variable called `@results`... I'd move this to a plain old variable assignment

Reply:
It is exposing `results` into the view.

Reply:
So why not use `@results = results`, which I find way more understandable of what I can use in the view?

Reply:
I needed to use `expose` so that tests can see this controller attribute.

Reply:
Wouldn't it be the same as `@results = results`?